frontend/src/index.css

```diff
@@ -212,7 +212,7 @@ input[type="number"] {
 }
 
 .basic-button {
-  background-color: hsl(84deg 55% 33%);
+  background-color: hsl(84deg 55% 22%);
```

Comment:
We do use a color scheme for this project with predefined shades, which we should use here as well. I don't think you've seen it yet, but I just sent you an invite to a Figma team, that's where we keep a sketch of the page and also the color scheme. If you want to make this button darker, the next shade in the scheme would probably be hsl(90deg 55% 23%) .

Reply:
Thanks! I will test that, if it is enough for the contrast checker.